src/views/PaymentConfirmation.vue

```diff
@@ -3,30 +3,43 @@
     <template #main>
       <div className="grid grid-rows-5 h-full place-items-center w-full">
         <div className="h-full w-full text-center grid grid-rows-2 font-semibold text-xl">
-          <div>{{ $t("symbol") }}</div>
+          <div>{{ $t('symbol') }}</div>
           <div>{{ paymentStore.firstName }}</div>
         </div>
         <div class="row-span-2 text-6xl font-bold w-fit h-full relative">
-          <div class="rounded-full absolute h-12 w-12 fill-white right-0 top-0 place-items-center grid" :class="{ 'bg-red-600': isConfirmed, 'bg-green-600': !isConfirmed}">
-            <IconCheckmark v-if="!isConfirmed"/>
-            <IconCross v-else/>
+          <div
+            class="rounded-full absolute h-12 w-12 fill-white right-0 top-0 place-items-center grid"
+            :class="{ 'bg-red-600': isConfirmed, 'bg-green-600': !isConfirmed }"
+          >
+            <IconCheckmark v-if="!isConfirmed" />
+            <IconCross v-else />
           </div>
-          <img class="rounded-full h-44 w-44 object-cover customborder border-4" alt="Titelblatt"
-            src="../assets/Titelseite.jpg">
+          <img
+            class="rounded-full h-44 w-44 object-cover customborder border-4"
+            alt="Titelblatt"
+            :src="
+              settStore.settings.MainItemImage
+                ? apiUrl + settStore.settings.MainItemImage
+                : '/Titelseite.jpg'
+            "
```

Comment:
The code is "show Titleseite.jpg if the mainItemImage isn't set", have you set a MainItemImage?

Reply:
@lebe1 ?

Reply:
Nope I did not. Maybe we should implement a default MainItemImage in the backend for developing purposes then?

Reply:
Works fine! So resolving this one